### ISSUE TITLE
[HW1] Form elements do not have associated labels on Messages page

### ISSUE THREAD

**Issue by prithviokade** [CMU-313/fall-2021-hw2-451-unavailable-for-legal-reasons#37]
The Accessibility score for the Messages page is 83. One of the warnings is that a form element does not have associated labels, so screen readers and other assistive technology may have difficulty announcing the element (this is the Lighthouse-reported problem being targeted).

**Comment by prithviokade** [comment]
Merged hw1 fix!
